### ISSUE TITLE
Windows path broken to csound diskin2 opcode

### ISSUE THREAD

**Issue by Leofltt** [TheSoundOfAIOSR/rg_production#10]
The path to the sample to be played appears without separators on windows.

Me and Fernando are working on a fix, but if that doesn't happen I will change the csound sampler to work in a different way where it doesn't need  a path. (I will explain more on this if we see that we can't find a workaround to fix the path issue in any way)

**Comment by fergarciadlc** [comment]
There was an issue with windows path separators, when we were passing the path with `"\"` or `"\\"` csound did not get the separators. To fix this, I made some tests and figured out that we can pass the path as with standard Unix separators `/` to ctcsound on windows and it detects the path correctly and playback the sample.

I also changed the path handling to pure `os.path`, the concatenations can be done with either `os.path.join` (and later replace `\\` for `/` on windows) or with normal string concatenation.

**Comment by fergarciadlc** [comment]
On commit 291baf818d42e3c89dca215edce3797bb541acc8:

There was an issue with windows path separators, when we were passing the path with `\` or `\\` csound did not get the separators. To fix this, I made some tests and figured out that we can pass the path as with standard Unix separators `/` to ctcsound on windows, it detects the path correctly and playback the sample.

I also changed the path handling to pure `os.path`, the concatenations can be done with either `os.path.join` (and later replace `\\` for `/` on windows) or with normal string concatenation.
